src/cli/backup/backup_test.go

```diff
@@ -66,3 +68,30 @@ func (suite *BackupUnitSuite) TestGenericDetailsCore() {
 	assert.NoError(t, err, clues.ToCore(err))
 	assert.ElementsMatch(t, expected, output.Entries)
 }
+
+func (suite *BackupUnitSuite) TestGenericDetailsCore_empty() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	bg := testdata.VersionedBackupGetter{
+		Details: &details.Details{
+			DetailsModel: details.DetailsModel{
+				Entries: []details.Entry{},
+			},
+		},
+	}
+
+	sel := selectors.NewExchangeBackup([]string{"user-id"})
+	sel.Include(sel.AllData())
+
+	_, err := genericDetailsCore(
+		ctx,
+		bg,
+		"backup-ID",
+		sel.Selector,
+		control.DefaultOptions())
+	require.Error(t, err, "has error")
+	assert.ErrorContains(t, err, "no items in backup", clues.ToCore(err))
```

Comment:
minor: error message checks are brittle.  An error sentinel and ErrorIs check would be more resilient.